docs/sql-migration-guide.md

```diff
@@ -40,6 +40,10 @@ license: |
   - In Spark 3.2, script transform default FIELD DELIMIT is `\u0001` for no serde mode, serde property `field.delim` is `\t` for Hive serde mode when user specifies serde. In Spark 3.1 or earlier, the default FIELD DELIMIT is `\t`, serde property `field.delim` is `\u0001` for Hive serde mode when user specifies serde.
 
   - In Spark 3.2, the auto-generated `Cast` (such as those added by type coercion rules) will be stripped when generating column alias names. E.g., `sql("SELECT floor(1)").columns` will be `FLOOR(1)` instead of `FLOOR(CAST(1 AS DOUBLE))`.
+  
+  - In Spark 3.2, the output schema of `SHOW TABLES` becomes `namespace: string, tableName: string, isTemporary: boolean`. In Spark 3.1 or earlier, the output schema of `SHOW TABLES` is `database: string, tableName: string, isTemporary: boolean` for the builtin catalog, is `namespace: string, tableName: string` for the v2 catalogs.
+  
+  - In Spark 3.2, the output schema of `SHOW TABLE EXTENDED` becomes `namespace: string, tableName: string, isTemporary: boolean, information: string`. In Spark 3.1 or earlier, the output schema of `SHOW TABLES` is `database: string, tableName: string, isTemporary: boolean, information: string` for the builtin catalog, is `namespace: string, tableName: string, isTemporary: boolean, information: string` for the v2 catalogs.
```

Comment:
if the only difference is `database`, maybe we can simplify it
```
In Spark 3.1 or earlier, the `namespace` field was named `database` for the builtin catalog, and no change for the v2 catalogs.
```

Reply:
OK